seaborn/categorical.py

```diff
@@ -2870,3 +3098,193 @@ def factorplot(x=None, y=None, hue=None, data=None, row=None, col=None,
+        data = x
+        x = None
+        warn = True
+
+    if "vals" in kwargs:
+        x = kwargs.pop("vals")
+        warn = True
+
+    if "groupby" in kwargs:
+        y = x
+        x = kwargs.pop("groupby")
+        warn = True
+
+    if "vert" in kwargs:
+        vert = kwargs.pop("vert", True)
+        if not vert:
+            x, y = y, x
+        orient = "v" if vert else "h"
+        warn = True
+
+    if "names" in kwargs:
+        kwargs.pop("names")
+        warn = True
+
+    if "join_rm" in kwargs:
+        kwargs.pop("join_rm")
+        warn = True
+
+    msg = ("The Seaborn categorical API has been changed. Attempting to adjust your "
+           "arguments for the new API (which might not work). Please update "
+           "your code. See the version 0.6 release notes for more info.")
+
+    if warn:
+        warnings.warn(msg, UserWarning)
+
+    plotter = _LVPlotter(x, y, hue, data, order, hue_order,
+                          orient, color, palette, saturation,
+                          width, k_depth, linewidth, box_widths)
+
+    if ax is None:
+        ax = plt.gca()
+
+    plotter.plot(ax, kwargs)
+    return ax
+
+lettervalueplot.__doc__ = dedent("""\
+    Create a letter value plot
+
+    Letter value (LV) plots are non-parametric estimates of the distribution of
+    a dataset, similar to boxplots. LV plots are also similar to violin plots
+    but without the need to fit a kernel density estimate. Thus, LV plots are fast
+    to generate, directly interpretable in terms of the distribution of data, and
+    easy to understand. For a more extensive explanation of letter value plots
+    and their properties, see Hadley Wickham's excellent paper on the topic:
+
+    http://vita.had.co.nz/papers/letter-value-plot.html
+
+    {main_api_narrative}
+
+    Parameters
+    ----------
+    {input_params}
+    {categorical_data}
+    {order_vars}
+    {orient}
+    {color}
+    {palette}
+    {saturation}
+    {width}
+    k_depth : "proportion" | "tukey" | "trustworthy", optional
+        The number of boxes, and by extension number of percentiles, to draw.
+        All methods are detailed in Wickham's paper. Each makes different
+        assumptions about the number of outliers and leverages different
+        statistical properties.
+    {linewidth}
+    box_widths : "linear" | "exponential" | "area"
+        Method to use for the width of the letter value boxes. All give similar
+        results visually. "linear" reduces the width by a constant linear factor,
+        "exponential" uses the proportion of data not covered, "area" is
+        proportional to the percentage of data covered.
+    p : float, optional
+        Proportion of data believed to be outliers. Is used in conjuction with
+        k_depth to determine the number of percentiles to draw. Defaults to 8
+        outliers.
+    {ax_in}
+    kwargs : key, value mappings
+        Other keyword arguments are passed through to ``plt.boxplot`` at draw
```

Comment:
Robotic copy paste. The keywords are actually only going to `plt.plot`. I think I'll change this so it goes to `plt.plot` and `plt.scatter`.